src/structures/ApplicationCommand.js

```diff
@@ -77,6 +77,12 @@ class ApplicationCommand extends Base {
      * @type {boolean}
      */
     this.defaultPermission = data.default_permission;
+
+    /**
+     * Autoincrementing version identifier updated during substantial record changes
+     * @type {Snowflake}
+     */
+    this.version = data.version;
```

Comment:
This should be checked in equals too

Reply:
sure

Reply:
Why should it? You can use equals with ApplicationCommandData and the version is something only Discord provides, you cannot have a version in your command data

Reply:
Equals takes an ApplicationCommand too

Reply:
Yeah I know that, that’s exactly why I made that comment because just comparing them like it was initially done was pretty stupid, but doing a check like the id check is better